#Use: Start exception handling block

try:
    print(10 / 0)
except ZeroDivisionError:
    print("Cannot divide by zero")


"""
10. except

Use: Catch exceptions in try block

Example: (see above)

"""